tests/cicd/main_build.yml

```diff
@@ -49,11 +49,16 @@ jobs:
     name: deepseismicagentpool
   steps:
   - bash: |
-      echo "Starting unit tests"
+      echo "Starting cv_lib unit tests"
       source activate seismic-interpretation
       pytest --durations=0 cv_lib/tests/
-      echo "Unit test job passed"
-    
+      echo "cv_lib unit test job passed"
+      echo ""
+      echo "Starting scripts unit tests"
+      source activate seismic-interpretation
```

Comment:
Why are we doing this twice?

Reply:
it's 2 tests cv_lib and scripts. I added a new build job as per your other comment.